web/themes/custom/server_theme/templates/server-theme-person-card.html.twig

```diff
@@ -0,0 +1,42 @@
+{% set extra_classes = [] %}
+
+{% if phone %}
+  {% set extra_classes = [
+    'grid',
+    'grid-flow-row',
+    'grid-cols-2',
+    'divide-x-2',
+    'divide-gray-200',
+    'divide-solid'
+  ] %}
+{% endif %}
+
+<div class="relative rounded-lg border flex flex-col justify-between shadow-lg max-w-[384px] bg-white">
```

Comment:
The twig file doesn't seem to be aligned with the rest of the code base, that uses composition (I'd expect to see composition inside the `ThemeTrait` you've created)

Reply:
@amitaibu Can you please explain "that uses composition (I'd expect to see composition inside the ThemeTrait". I didn't quite get it. Is there are any examples that I can refer to?

Reply:
For example https://github.com/Gizra/drupal-starter/blob/5f18634af73986c887942354fe3c533b09b167fc/web/modules/custom/server_general/src/ThemeTrait/ElementNodeNewsThemeTrait.php#L48

Reply:
Thanks, you can remove the file. Can you post new screenshots please, of how it looks